test/madnlp_dense.jl

```diff
@@ -181,12 +182,10 @@ function _compare_dense_with_sparse(n, m, ind_fixed)
     )
 
     nlp = build_qp_test(; n=n, m=m, fixed_variables=ind_fixed)
-    x, l = copy(nlp.x), copy(nlp.l)
-
     ips = MadNLP.Solver(nlp, option_dict=sparse_options)
     MadNLP.optimize!(ips)
 
-    # Reinit NonlinearProgram to avoid side effect
+    # Reinit DenseQP to avoid side effect
```

Comment:
Here the issue was that `nlp.x`, `nlp.l` ... were updated at the end of the solve. Hence, restarting the resolution from the same `NonlinearProgram` wamstarted implicitly the solver. Maybe this is not the case anymore (in my opinion, the model should be immutable)